preamble.tex

```diff
@@ -14,7 +14,7 @@
 %--------------------------------------+
 % Language, hyphenation, encoding, etc |
 %--------------------------------------+
-\usepackage[spanish,english]{babel}
+\usepackage[spanish,english]{biblatex}
```

Comment:
Yes this was not included in the `real` commit. That line was directly commented. 

p.s.: the `real` part comes from the inadvertent choice of githubs' complicated fork--> PR --> review PR --> approve PR .. process. I had to manually edit files online in the third step, missing this edit that was done locally.

if you know better how to commit to an open Fork PR without push access, be welcomed :)